app/javascript/styles/basics.scss

```diff
@@ -1,6 +1,6 @@
 body {
   font-family: 'Roboto', sans-serif;
-  background: $color1 image-url('background-photo.jpg');
+  background: $color1 url('../images/background-photo.jpg');
```

Comment:
It seems weird that the stylesheets move into `app/javascript/...` -- is that normal for webpack?